devices.js

```diff
@@ -3081,8 +3081,10 @@ const devices = [
         zigbeeModel: ['BoxDIM2 98425031'],
         model: '98425031',
         vendor: 'Nordtronic',
-        description: 'Box Dimmer 2.0',
```

Comment:
The descriptions should match the one advertised by the manufacturer, e.g. https://nordtronic.dk/images/Manual_BoxDimmer2_98425031.pdf, so `Box Dimmer 2.0` was the correct description here.

Reply:
I've check all the other names with the documentation of the suppliers. They should be correct now.